aiosmtpd/tests/conftest.py

```diff
@@ -99,6 +104,22 @@ def cache_fqdn(session_mocker: MockFixture):
 # region #### Common Fixtures #########################################################
 
 
+def _server_resource(name: str, /) -> Generator[Path, None, None]:
+    ref = importlib_resources.files("aiosmtpd.tests.certs") / name
```

Comment:
Weird this is shipped in wheels.. I'd go for a relative path myself.

OTOH, I recommend integrating https://trustme.rtfd.io like I did in aiohttp and many other places so that the TLS certificates are ephemeral and don't even exist in the repo.

Reply:
I agree it is possible, however for me it looks like it is out of scope for this PR.